http.go

```diff
@@ -21,6 +22,13 @@ func (s *Server) Handler() http.Handler {
 
 	mux.Handle(pat.Post("/import"), handleImport(s))
 
+	// TODO match without trailing slash as well
```

Comment:
nitpick: move the comment down to the `/*` route since that's the route it's referring to